packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js

```diff
@@ -77,7 +77,7 @@ export class CheckoutDeliveryOption extends PureComponent {
 
     renderPrice() {
         return (
-            <strong>
+            <strong style={ { display: 'inline-block' } }>
```

Comment:
Why not just add this in style? In-lining prevents for further modification in parent themes.